src/specificAnimalClasses/SnakeClass.ts

```diff
@@ -3,7 +3,7 @@ import Reptile from "../interfaces/IReptile";
 import { ICanSwim } from "../interfaces/ICanSwim";
 import { ICanWalk } from "../interfaces/ICanWalk";
 import { ageToBeConsideredAdult, snakes } from "../constVariables";
-import { checkAge, checkLength } from "../verificationFunction/inputCheck";
+import { checkAge, checkLength, checkNameNotEmpty } from "../verificationFunction/inputCheck";
```

Comment:
What is the need for AgeBracket class ? can we go with a simple  isAdult: boolean  field instead ? if not  - why ?